blazor/tabs/how-to/prevent-content-swipe-selection.md

```diff
@@ -9,43 +9,37 @@ documentation: ug
 
 # Prevent content swipe selection in Blazor Tabs Component
 
-The tab selection can be prevented on touch swipe action by using the Tab [Selecting](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.TabEvents.html#Syncfusion_Blazor_Navigations_TabEvents_Selecting) event.
+The [`SwipeMode`](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfTab.html#Syncfusion_Blazor_Navigations_SfTab_SwipeMode) property in the Tab component enables users to navigate between tabs using swipe gestures, whether through touch or mouse inputs. This feature enhances the user experience by providing an intuitive way to switch tabs.
```

Comment:
The SwipeMode property in the Tab component allows users to navigate between tabs using swipe gestures, improving the overall navigation experience. It supports both touch and mouse inputs, enabling intuitive tab switching.

However, in certain cases, such as when a tab contains critical elements like a form, accidental swipes can disrupt the user's workflow. For example, if a user is filling out a form and mistakenly swipes, the tab might change unexpectedly, potentially leading to unsaved data or interruptions.

To address this, developers can customize the SwipeMode property based on the needs of the application. By adjusting this setting, developers can balance smooth, intuitive navigation with protecting the user experience and safeguarding against accidental tab switches.

The following are the available SwipeMode options for the Tab component:

[TabSwipeMode.Touch & TabSwipeMode.Mouse](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.TabSwipeMode.html) – By default, this option allows the user to swipe between tabs using both touch and mouse actions.
[TabSwipeMode.Touch](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.TabSwipeMode.html) – This option allows users to swipe between tabs using touch gestures only.
[TabSwipeMode.Mouse](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.TabSwipeMode.html) – This option allows users to swipe between tabs using mouse gestures only.
[~TabSwipeMode.Touch & ~TabSwipeMode.Mouse](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.TabSwipeMode.html) – This disables both touch and mouse swipe actions, preventing any unintended tab switches.